docs/device-api.md

```diff
@@ -0,0 +1,109 @@
+| `PartialNetworkInterface` | iface_id              |   O      |   O            |   O          | **R**      |   O          |
+|                           | rx_rate_limiter       |   O      |   O            |   O          | **R**      |   O          |
+|                           | tx_rate_limiter       |   O      |   O            |   O          | **R**      |   O          |
+| `RateLimiter`             | bandwidth             |   O      |   O            | **R**        | **R**      |   O          |
+|                           | ops                   |   O      |   O            | **R**        | **R**      |   O          |
+| `TokenBucket`             | one_time_burst        |   O      |   O            | **R**        | **R**      |   O          |
+|                           | refill_time           |   O      |   O            | **R**        | **R**      |   O          |
+|                           | size                  |   O      |   O            | **R**        | **R**      |   O          |
+| `Vm`                      | state                 |   O      |   O            |   O          |   O        |   O          |
+| `Vsock`                   | guest_cid             |   O      |   O            |   O          |   O        | **R**        |
+|                           | uds_path              |   O      |   O            |   O          |   O        | **R**        |
+|                           | vsock_id              |   O      |   O            |   O          |   O        | **R**        |
+
+## Output Schema
+
+| Schema                  | Property          | keyboard | serial console | virtio-block | virtio-net | virtio-vsock |
+|-------------------------|-------------------|:--------:|:--------------:|:------------:|:----------:|:------------:|
+| `Error`                 | fault_message     |   O      |   O            |   O          |   O        |   O          |
+| `InstanceInfo`          | app_name          |   O      |   O            |   O          |   O        |   O          |
+|                         | id                |   O      |   O            |   O          |   O        |   O          |
+|                         | state             |   O      |   O            |   O          |   O        |   O          |
+|                         | vmm_version       |   O      |   O            |   O          |   O        |   O          |
+| `MachineConfiguration`  | cpu_template      |   O      |   O            |   O          |   O        |   O          |
+|                         | ht_enabled        |   O      |   O            |   O          |   O        |   O          |
+|                         | mem_size_mib      |   O      |   O            |   O          |   O        |   O          |
+|                         | track_dirty_pages |   O      |   O            |   O          |   O        |   O          |
+|                         | vcpu_count        |   O      |   O            |   O          |   O        |   O          |
+
+## Instance Actions
```

Comment:
For consistency sake, I would add again the swagger file reference here.

Reply:
Done. Incoming.